src/AreaChart.js

```diff
@@ -153,6 +159,7 @@ export default class AreaChart extends React.Component {
     }
 
     areaGenerator
+      .curve(curve)
```

Comment:
is there an ability to turn it off? If I'm reading this correctly it will curve as a default, we probably want that behind a flag